src/extension/experimentService/experimentsStrings.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as nls from "vscode-nls";
+const localize = nls.loadMessageBundle();
```

Comment:
Please take into account that on preview version we also need to apply nls.config